subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -720,6 +720,30 @@
                 "Method has been removed"
             ]
         },
+        {
```

Comment:
@octylFractal I gave up on this too.

@tresat weirdly, when I downloaded the "accept all errors" file, this wasn't sorted properly and I had to resort it.

Reply:
That's weird.  If that happens again can you quick send me what API change you made so I can investigate.